php/antidot/composer.json

```diff
@@ -0,0 +1,87 @@
+{
+  "name": "antidot-fw/reactive-starter",
+  "description": "Reactive Antidot Framework Apllication Starter",
+  "type": "project",
+  "license": "BSD-2-Clause",
+  "authors": [
+    {
+      "name": "kpicaza"
+    }
+  ],
+  "repositories": [
+    {
+      "type": "vcs",
+      "url": "https://github.com/kpicaza/psr11-monolog.git"
+    }
+  ],
+  "require": {
+    "php": "^7.4|^8.0",
+    "ext-json": "*",
+    "antidot-fw/react-framework": "^1.0.0",
+    "antidot-fw/fast-router-adapter": "^0.1.0",
+    "antidot-fw/cli": "^1.1.0",
+    "antidot-fw/container": "^0.1.0",
+    "antidot-fw/dev-tools": "^0.1.2",
+    "antidot-fw/event-dispatcher": "^2.0.3",
+    "antidot-fw/framework": "^0.1.1",
+    "antidot-fw/logger": "^1.1.0",
+    "antidot-fw/symfony-config-translator": "^1.1.0",
+    "antidot-fw/yaml-config-provider": "^0.1.0",
+    "webmozart/assert": "^1.7.0",
+    "wshafer/psr11-monolog": "^3.0.0|@dev"
+  },
+  "require-dev": {
+    "franzl/whoops-middleware": "^2.0",
+    "phpro/grumphp": "^1.0",
+    "phpstan/phpstan": "^0.12",
+    "phpunit/phpunit": "^8.0|^9.0",
+    "roave/security-advisories": "dev-master",
+    "squizlabs/php_codesniffer": "^3.4",
+    "symfony/var-dumper": "^5.2",
+    "laminas/laminas-component-installer": "^2.1.2"
```

Comment:
dev dependencies are not necessary and are recommended to be removed

`php/antidot/public/favicon.ico` This one is also

Reply:
Thanks! maybe will be correct to commit the cached config to avoid generating it in the first request?

Reply:
not needed. the warm-up phase will do this for you